scc/src/main.rs

```diff
@@ -24,7 +23,10 @@ mod opts;
 
 const BUNDLE_FILE_NAME: &str = "final.redscripts";
 const BACKUP_FILE_NAME: &str = "final.redscripts.bk";
-const TIMESTAMP_FILE_NAME: &str = "redscript.ts";
+const OLD_TIMESTAMP_FILE_NAME: &str = "redscript.ts";
```

Comment:
I'd name it `legacy` instead of old to clarify it's for backwards compatibility
```suggestion
const LEGACY_TIMESTAMP_FILE_NAME: &str = "redscript.ts";
```